packages/gatsby-recipes/src/providers/fs/directory.test.js

```diff
@@ -0,0 +1,16 @@
+const directory = require(`./directory`)
+const resourceTestHelper = require(`../resource-test-helper`)
+
+const root = __dirname
+
+describe(`directory resource`, () => {
+  test(`e2e directory resource test`, async () => {
+    await resourceTestHelper({
+      resourceModule: directory,
+      resourceName: `Directory`,
+      context: { root },
+      initialObject: { path: `src/images` },
+      // partialUpdate: { content: content + `1` },
```

Comment:
this should be uncommented and be something like `partialUpdate: { path: `src/images2` },`

Reply:
My local does not reflect this:
![image](https://user-images.githubusercontent.com/25191/81880165-bbe28000-955a-11ea-995f-7a87de49bb3f.png)

But is probably why it's a good idea not to `git force push` anything. Kids these days... :old_man_shaking_stick:

Reply:
The `path` in the image above caused a stray `src` directory to be left behind. Test has been updated to fix this.